directoryhash.bash

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# directoryhash.bash generates a sha256 hash of the contents of a directory
```

Comment:
Bash scripts should be in `make/buf-push-action/scripts` and have a corresponding `make` target

Reply:
The two *.bash scripts in the root are there because they are called when the action runs. I'll try to get them working with make.

Reply:
Ah - I might make a `scripts` directory at the top-level then